examples/js/loaders/ColladaLoader.js

```diff
@@ -1624,7 +1624,15 @@ THREE.ColladaLoader.prototype = {
 
 		function getCamera( id ) {
 
-			return getBuild( library.cameras[ id ], buildCamera );
+			var data = library.cameras[ id ];
+
+			if ( data !== undefined ) {
```

Comment:
Actually, considering that the check is simple, what do you think about logging a warning in the console for these situations? Something like...

```js
var data = library.cameras[ id ];

if ( data !== undefined ) {

    return getBuild( data, buildCamera );

}

console.warn( 'THREE.ColladaLoader: Couldn't find camera ', id );
return null;
```

We can later also warn the user that the file doesn't follow the collada spec. By now it serves to document the fact that we're being tolerant, and also to highlight "broken" files. It will also be handy in the future if someone is having issues with a collada file and we see that in the logs, we can already suspect that they user a bad exporter.

Reply:
Great idea 👍 . I'll make a PR 😊